src/modules/LanItems.rb

```diff
@@ -2018,21 +2027,24 @@ def Commit
         newdev["INTERFACETYPE"] = @type
       end
 
-      NetworkInterfaces.Name = Ops.get_string(@Items, [@current, "ifcfg"], "")
-      NetworkInterfaces.Current = deep_copy(newdev)
+      if GetCurrentMap() != newdev
+        NetworkInterfaces.Name = Items()[@current]["ifcfg"] || ""
+        NetworkInterfaces.Current = deep_copy(newdev)
```

Comment:
The above 3 lines are still a mystery to me :-/ I'll revisit them after I review the rest.